### DISCUSSION TITLE
Changing a folder for storing planer files

### DISCUSSION THREAD

**Discussion by GeorgeRub** [ivan-lednev/obsidian-day-planner#554] [Q&A] [answered]
Hi! I could be awesome to have ability to change folder for holding all planer files. I see that all files are collecting in the root folder. It will create a huge number of files and make it overweight for managing it in the future.

**Comment by ivan-lednev** [comment, marked as answer]
Hey! You can use core daily notes plugin to set the folder where these notes go.

**Comment by mataide** [comment]
What daily notes plugin? There is many...

![image](https://github.com/user-attachments/assets/f9fef5cc-cef5-4b59-b87a-0d5f913042ca)

**Comment by ivan-lednev** [comment]
These are community plugins. There is only one core daily notes plugin:
![image](https://github.com/user-attachments/assets/45f02ecb-b992-4b5f-b993-4645c13e771f)

You can do the same with the community periodic notes plugin.
